app/src/main/java/com/simplecity/amp_library/utils/PlaylistUtils.java

```diff
@@ -216,8 +216,13 @@ public static void clearMostPlayed() {
         void onSave(Playlist playlist);
     }
 
-    public static void makePlaylistMenu(SubMenu sub) {
-        DataManager.getInstance().getPlaylistsRelay()
+    public static void makePlaylistMenu(SubMenu sub, Observable<Long> stopEventTrigger, boolean autoUpdate) {
```

Comment:
It looks like this was just an oversight - the Observable remains subscribed and listening for emissions after we've finished making our playlist menu. We only need the first emission from the playlist relay.

Using the stopEventTrigger/LifecycleCallbacks seems like overkill here. I would think instead, we should just be calling `playlistListObservable.first(Collections.emptyList())`, which ensures the `Observable` only emits once and then is disposed.